keyboards/mechlovin/zed65/rev1/keymaps/default/keymap.c

```diff
@@ -16,13 +16,45 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
 #include QMK_KEYBOARD_H
+#include "rev1.h"
+
+enum custom_keycodes {
+    RGB_TOGGLE_LOGO = QK_KB_0,
```

Comment:
This isn't a keyboard-level keycode, so should use the user range.
```suggestion
    RGB_TOGGLE_LOGO = QK_USER_0,
```